.vscode/settings.json

```diff
@@ -82,5 +80,29 @@
       "language": "markdown",
       "scheme": "file"
     }
-  ]
+  ],
+  "workbench.colorCustomizations": {
```

Comment:
Was this meant to be added?

Reply:
Yeah I figure if it's not there already, nobodoy will have problem with it. Wdyt?

Reply:
I think theme colors are not required to the project as this is customized in the local editors